src/core/services/gsp_gpu.cpp

```diff
@@ -261,6 +261,18 @@ void GPUService::setAxiConfigQoSMode(u32 messagePointer) {
 	mem.write32(messagePointer + 4, Result::Success);
 }
 
+void GPUService::setBufferSwap(u32 messagePointer) {
```

Comment:
This would make games crash via err:f since it does not handle the IPC response header or the IPC result code. I am not sure what game to use to test it though :/

If there's none we know of atm, I will keep the code but add a Helpers::panic to revisit this when a game like that is found.

Reply:
Fixed the IPC buffer for SetBufferSwap, still added a panic for when there's a known test case